ast/src/lang/call_finder.rs

```diff
@@ -66,6 +67,16 @@ pub fn func_target_file_finder<G: Graph>(
         return Some(tf);
     }
 
+    // Fifth try: If operand exists, try operand-based resolution
+    if let Some(ref operand) = operand {
+        if let Some(target_file) = find_function_with_operand(operand, func_name, graph) {
```

Comment:
special case for typescript: if the fn starts with `use`, then search for the OPERAND directly as a function 

that will create a CALLS edge from the test or component to the hook

(this is for tests only???)